src/Raven.Server/Config/Categories/LicenseConfiguration.cs

```diff
@@ -46,5 +46,16 @@ public class LicenseConfiguration : ConfigurationCategory
         [DefaultValue(false)]
         [ConfigurationEntry("License.SkipLeasingErrorsLogging", ConfigurationEntryScope.ServerWideOnly)]
         public bool SkipLeasingErrorsLogging { get; set; }
+        
+        
+        [Description("EXPERT ONLY. Disable automatic updating of the license")]
+        [DefaultValue(false)]
+        [ConfigurationEntry("License.DisableAutoLicenceUpdate", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
There is a typo here, but I think it would be better to change this to `License.DisableAutoUpdate`. Thoughts?

Reply:
done